README.md

```diff
@@ -0,0 +1,17 @@
+#data-structures
+
+###This repository holds sample code for practicing TCP/IP concepts.
+
+*Feb 9th, 2015* - Added scripts for a basic echo server and client:
+
+Collaborations:
+Constantine Hatzis
+
+
+Resources:
+<ul>
+<li>Cris Ewing</li>
+<li>stackoverflow.com</li>
+<li>tutorialspoint.com/python</li>
+<li>docs.python.org</li>
```

Comment:
And which docs, exactly.  Make these lists real links like so:

* [google](http://google.com)
* [python docs](http://docs.python.org)